charts/schema-loading/templates/batchjob-schema-loading.yaml

```diff
@@ -55,6 +56,18 @@ spec:
         - "-j"
         - "{{ .Values.schemaLoading.contactPoints }}"
         {{- end }}
+        {{- else }}
+        - "--config"
+        - "database.properties"
+        {{- if eq .Values.schemaLoading.schemaType "ledger" }}
+        - "--coordinator"
+        {{- end }}
```

Comment:
If `Schema Type` is `ledger`, it added the `--coordinator` flag automatically.

Reply:
@feeblefakie the coordinator table is created only when `--coordinator` is specified.

Reply:
Oh, OK.
Hmm, I think it might be confusing.
Users usually don't care if they need to create a coordinator or not, so shouldn't the `--coordinator` option be specified automatically if there is `transaction: true` in a schema file?
Or, is there any case where the automation goes wrong?

Reply:
@feeblefakie Users need to care the coordinator table when they use the multi-storage transactions or the 2PC transactions because in that case, users need to specify in which database they create the coordinator table.

Reply:
Ah, right.